### DISCUSSION TITLE
are you going to publish distributables on Github?

### DISCUSSION THREAD

**Discussion by raydac** [microsoft/openjdk#26] [Q&A] [answered]
Hi
I could add support for the OpenJDK distributive into my maven plugin [mvn-jlink](https://github.com/raydac/mvn-jlink),are you going to publish JDK distributives on Github?

**Comment by karianna** [comment]
Hi @raydac - Are you meaning can you have access to an API like AdoptOpenJDK?

**Comment by raydac** [comment, marked as answer]
yes, or may be some plans for publishing of binary distributives as releases on github?

**Reply by karianna** [reply]
OK, I understand the requirement now.  No guarantees on this but we completely understand that folks will want API and/or GitHub access to our binaries.  We'll discuss this further internally for sure!
